src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder readTextBuilder = getTextFromFile(fromFileName);
+        long supply = 0;
+        long buy = 0;
+        long result = 0;
+        String[] optimizationText = String.valueOf(readTextBuilder).split(System.lineSeparator());
+        for (String cloud : optimizationText) {
+            String[] cutText = cloud.split(" ");
+            if (cutText[0].equals("supply")) {
+                supply = supply + Integer.parseInt(cutText[1]);
+            } else if (cutText[0].equals("buy")) {
+                buy = buy + Integer.parseInt(cutText[1]);
+            }
+        }
+        readTextBuilder = new StringBuilder();
+        if (supply > buy) {
+            result = supply - buy;
+        } else if (supply < buy) {
+            result = buy - supply;
+        }
+        readTextBuilder
+                .append("supply,").append(supply).append(System.lineSeparator())
+                .append("buy,").append(buy).append(System.lineSeparator())
+                .append("result,").append(result);
+        String text = String.valueOf(readTextBuilder);
```

Comment:
move this to a separate method
add constants for indexes

Reply:
it is fixed